src/components/calcite-dropdown-group/calcite-dropdown-group.tsx

```diff
@@ -104,8 +111,17 @@ export class CalciteDropdownGroup {
       ) : null;
 
     return (
-      <Host role="menu" scale={scale} title={this.groupTitle}>
-        <div dir={dir}>
+      <Host role="menu">
+        <div
+          class={{
+            container: true,
```

Comment:
Can you clarify why the additional container class on this line? (`class="container container--s"`)

Reply:
Its there to be used in the CSS incase needed. The other classes are modifiers in BEM.